src/main/java/com/evilcorp/bloom/dto/CategoryDto.java

```diff
@@ -0,0 +1,11 @@
+package com.evilcorp.bloom.dto;
+
+import jakarta.validation.constraints.NotBlank;
+
+public class CategoryDto {
+
+  @NotBlank(message = "Category name cannot be blank")
+  public String category;
```

Comment:
Is this specified by the user? If yes, it is good to have some constraints for minimum and max size of the category name

Reply:
added validation for length